### ISSUE TITLE
Solution to library not automatically being included in iOS Podfile

### ISSUE THREAD

**Issue by nohns** [ammarahm-ed/react-native-simple-jsi#5]
Hi Ammar!

Thanks for the great work you have done here! 

While following your directions on your blog, I had an issue where the library pod was not automatically discovered when I created the new example project within the library directory. I figured out it was because this line was missing in `example/ios/podfile`:

`pod 'react-native-jsi-template', :path => '../..'`

I inserted it, ran `pod install` and everything is now discovered as it should. I don't know why it did not automatically add it, as it says in the blog it should. But you could maybe add this as a heads up for people that do not see their library show up in Xcode when they open their project. Just a suggestion, as it took a little bit of time for me to troubleshoot what was wrong :)

Greetings!

**Comment by mfbx9da4** [comment]
also just faced this issue would be good to update blog and podfile!

**Comment by ammarahm-ed** [comment]
@mfbx9da4 Reading this again, I have realized that the directions on the blog are correct. The difference is that you have to add your library via `npm install ../react-native-jsi-yourlibrary` first in your app project. Then pod will automatically detect your library and add it accordingly.

**Comment by mfbx9da4** [comment]
Yeah I also realized you were right but I was caught out by this because I hadn't done an `npm install` in the root of the 
project.

This point might be a good place to put it something like

> Ensure you have run "npm install" in the root of the project and under "example/"

![image](https://user-images.githubusercontent.com/1690659/150823672-026d554a-bf36-48c8-9221-90af573665a3.png)

**Comment by mfbx9da4** [comment]
Do you mention that step in the blog though?

![image](https://user-images.githubusercontent.com/1690659/151155671-fec3e3b9-6c75-49f6-8d93-405a6d1fdea3.png)


Shouldn't you say 

> Now add "react-native-jsi-yourlibrary": ".." to example/package.json

**Comment by ammarahm-ed** [comment]
@mfbx9da4 Delete the example and init a new example app because it used older version of RN at the time of writing.

**Comment by chanphiromsok** [comment]
<img width="229" alt="Screen Shot 2023-07-19 at 5 00 18 PM" src="https://github.com/ammarahm-ed/react-native-jsi-template/assets/65760336/b9b09517-f1f8-4b70-96e7-c106835096ea">
I could not config CMakeList point to jni.h
